management/backup.py

```diff
@@ -473,10 +473,91 @@ def list_target_files(config):
 			raise ValueError(e.reason)
 
 		return [(key.name[len(path):], key.size) for key in bucket.list(prefix=path)]
-
+	elif target.scheme == "b2":
+		api = b2_perform_authentication(target.username, target.password)
+		if api == None:
+			raise ValueError("B2 authentication failed for unknown reason.")
+		(auth_token, api_url) = api
+
+		bucket_id = b2_get_bucket_id(target.username, api_url, auth_token, target.hostname)
+		return b2_list_files(target.username, api_url, auth_token, bucket_id, target.path)
 	else:
 		raise ValueError(config["target"])
 
+def b2_perform_authentication(acc_id, app_key):
```

Comment:
Adding a URL to the docs for each of these would be handy. https://www.backblaze.com/b2/docs/b2_authorize_account.html

Have you considered naming the method 'b2_authorize_account', so it's the same name?